android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -178,6 +205,15 @@ public CustomTwilioVideoView(ThemedReactContext context) {
         audioManager = (AudioManager) themedReactContext.getSystemService(Context.AUDIO_SERVICE);
         myNoisyAudioStreamReceiver = new BecomingNoisyReceiver();
         intentFilter = new IntentFilter(Intent.ACTION_HEADSET_PLUG);
+
+        // Create the local data track
+       // localDataTrack = LocalDataTrack.create(this);
+       localDataTrack = LocalDataTrack.create(getContext());
+
+       // Start the thread where data messages are received
+        dataTrackMessageThread.start();
```

Comment:
Is there any cleanup that needs to happen with this thread?

Reply:
Done onHostDestroy dataTrackMessageThread.quit();